tf_files/configs/kube-setup-fence.sh

```diff
@@ -62,12 +68,57 @@ if [[ -d "${WORKSPACE}/${vpc_name}_output" ]]; then # update secrets
     g3kubectl create secret generic fence-json-secret --from-file=fence_credentials.json=./apis_configs/fence_credentials.json
   fi
 
-  if ! g3kubectl get secrets/fence-jwt-keys > /dev/null 2>&1; then
-    g3kubectl create secret generic fence-jwt-keys --from-file=./jwt-keys
+  if ! kubectl get configmaps/projects > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/projects.yaml" ]]; then
+      touch "apis_configs/projects.yaml"
+    fi
+    kubectl create configmap projects --from-file=apis_configs/projects.yaml
+  fi
+
+  if ! kubectl get configmaps/user-dir > /dev/null 2>&1; then
+    mkdir -p ./apis_configs/user-dir
+    kubectl create configmap user-dir --from-file=./apis_configs/user-dir
+  fi
+
+  if ! kubectl get secrets/fence-jwt-keys > /dev/null 2>&1; then
+    kubectl create secret generic fence-jwt-keys --from-file=./jwt-keys
   fi
+  
+  if ! kubectl get configmaps/fence-sshconfig > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/.ssh/config" ]]; then
+        echo '''
+        Host squid.internal
+          ServerAliveInterval 120
+          HostName cloud-proxy.internal.io
+          User ubuntu
+          ForwardAgent yes
+
+        Host sftp.planx
+          ServerAliveInterval 120
+          HostName sftp.planx-pla.net
+          User foo
+          ForwardAgent yes
+          IdentityFile ~/.ssh/id_rsa
+          ProxyCommand ssh ubuntu@squid.internal nc %h %p 2> /dev/null
+
+        Host cloud-proxy.internal.io
+          StrictHostKeyChecking no
+          UserKnownHostsFile=/dev/null
+        ''' > ./apis_configs/.ssh/config
+    fi
+    kubectl create configmap fence-sshconfig --from-file=./apis_configs/.ssh/config
+  fi
+
+  if ! kubectl get secrets/fence-public-key > /dev/null 2>&1; then
```

Comment:
Can we combined fence-public-key and fence-ssh-key into one secret - fence-ssh-keys ?

Reply:
Fixed at 949c772f16eef81339337b42dbf68fb52f6ae903
Thanks